src/Func.cc

```diff
@@ -817,6 +864,39 @@ static int get_func_priority(const std::vector<AttrPtr>& attrs)
 	return priority;
 	}
 
+// Get a function's groups from its Scope's attributes. Errors if it sees any
+// problems. Less strict than get_func_priority()
```

Comment:
What does "less strict" mean here?

Reply:
I've updated the comment. get_func_priority() checks for non-applicable attributes for functions as a side-effect and I didn't want to duplicate that.